pkgs/development/python-modules/pgzero/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
```

Comment:
Using python3Packages is strictly forbidden in pythonPackages, use the package variables directly.